themes/default/content/docs/guides/adopting/import.md

```diff
@@ -248,7 +248,7 @@ var group = new SecurityGroup("my-sg",
 
 {{< /chooser >}}
 
-> **Note:** Import IDs are resource specific. The ID to use is the same as the ID that gets assigned when Pulumi has provisioned a resource of that type from scratch.
+> **Note:** Import IDs are resource specific. The ID to use is the same as the ID that gets assigned when Pulumi has provisioned a resource of that type from scratch and can be found in the resource's API docs. We support import IDs for all generally available providers.
```

Comment:
would you mind updating this to use the note styles instead of the quote styles?